app/style/CreateWalletForm.less

```diff
@@ -51,6 +51,12 @@
   top: 122px;
 }
 
+.red-error {
```

Comment:
and change this to `.seed-area .error`?

Just `red-error` seems a too generic name for a class with `position: fixed`.